requirements.txt

```diff
@@ -42,3 +42,4 @@ pilmoji==2.0.2
 imagehash==4.3.1
 hikari-toolbox==0.1.4
 google-generativeai==0.3.2
+grpcio==1.44.0
```

Comment:
The docker container currently has `grpcio==1.68.1`, it would be safer to pin to that. Could you please explain why we're explicitly adding this dependency at all though.